UPGRADE.md

```diff
@@ -27,7 +27,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 - check for usages of `TransportEditFormType` - it was removed and all it's attributes were moved to `TransportFormType` so use this form instead
 - check for usages of `PaymentEditFormType` - it was removed and all it's attributes were moved to `PaymentFormType` so use this form instead
 - check for usages of `ProductEditFormType` - it was removed and all it's attributes were moved to `ProductFormType` so use this form instead
-- pay attention to javascripts bound to your forms as well as the elements' names and ids has changed (e.g. from `#product_edit_form_productData` to `#product_form`) 
+- pay attention to javascripts bound to your forms as well as the elements' names and ids has changed (e.g. from `#product_edit_form_productData` to `#product_form`)
+
+### [shopsys/shopsys]
+- to use custom postgres configuration check changes in the `docker-compose.yml` templates and replicate them, there is a new volume for `postgres` container

```

Comment:
I would probably link "the changes" to this PR :slightly_smiling_face:

Reply:
I think in here should be information on that you need to run `docker-compose up -d` to take the changes in `docker-compose` into account